crates/red_knot/src/types.rs

```diff
@@ -46,6 +51,45 @@ impl Type {
     pub const fn is_unknown(&self) -> bool {
         matches!(self, Type::Unknown)
     }
+
+    // NOTE: naming? `get_member_type`
```

Comment:
I'm OK with simplified naming that leaves out `_type`; we understand that we are working with types here, everything is a type.